<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CountdownTimer</class>
 <widget class="QWidget" name="CountdownTimer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>197</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>197</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/icons/Yes, there are two rocket icons..png</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="timeLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>312</width>
       <height>88</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>312</width>
       <height>88</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Droid Sans Mono</family>
       <pointsize>48</pointsize>
      </font>
     </property>
     <property name="text">
      <string notr="true">00:00:00</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startStop">
       <property name="text">
        <string>&amp;Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reset">
       <property name="text">
        <string>&amp;Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSet">
       <property name="text">
        <string>Se&amp;t</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="preset1">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="preset2">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="preset3">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="preset4">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="preset5">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="preset6">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="preset7">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="preset8">
       <property name="text">
        <string>Preset</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>startStop</tabstop>
  <tabstop>reset</tabstop>
  <tabstop>btnSet</tabstop>
  <tabstop>preset1</tabstop>
  <tabstop>preset2</tabstop>
  <tabstop>preset3</tabstop>
  <tabstop>preset4</tabstop>
  <tabstop>preset5</tabstop>
  <tabstop>preset6</tabstop>
  <tabstop>preset7</tabstop>
  <tabstop>preset8</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>startStop</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>startStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reset</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSet</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>set()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>70</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset1</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>presetOne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset2</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>presetTwo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>161</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset3</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>presetThree()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>237</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset4</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>presetFour()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>177</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset5</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>presetFive()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset6</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>presetSix()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>29</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset7</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>presetSeven()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>preset8</sender>
   <signal>released()</signal>
   <receiver>CountdownTimer</receiver>
   <slot>presetEight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>startStop()</slot>
  <slot>reset()</slot>
  <slot>set()</slot>
  <slot>presetOne()</slot>
  <slot>presetTwo()</slot>
  <slot>presetThree()</slot>
  <slot>presetFour()</slot>
  <slot>presetSix()</slot>
  <slot>presetFive()</slot>
  <slot>presetSeven()</slot>
  <slot>presetEight()</slot>
  <slot>addMinute()</slot>
  <slot>subMinute()</slot>
  <slot>addSecond()</slot>
  <slot>subSecond()</slot>
 </slots>
</ui>
